### PR TITLE
Admin translation

### PR DESCRIPTION
**Pull request by ashish0910**
Added admin translation on changing language 

Current behaviour:
current

Expected behaviour:
result

### CODE REVIEW

File: dashboard/views/users.ejs [hunk 1]
```diff
@@ -38,11 +38,15 @@
...
-						<script>
-						window.addEventListener('localized', function() {  // HACK: Force localization of student combo
-							var selection = document.getElementsByClassName("select2-selection__rendered");
-							if (selection && selection[0] && selection[0].title == "Student") selection[0].innerHTML = document.webL10n.get("student");						});
-						</script>
+			<script>
+			window.addEventListener('localized', function() {  // HACK: Force localization of student combo
+				var selection = document.getElementsByClassName("select2-selection__rendered");
+				if (selection && selection[0] && typeof selection[0].innerHTML == "string") {
+                                var user = document.getElementById("user-type-select2");
```
Comment (NikhilM98) [inline dashboard/views/users.ejs]: There's still some indentation issue

Comment (NikhilM98) [reply dashboard/views/users.ejs]: The changes are not appearing here

**Comment by ashish0910** [comment]
@NikhilM98 i don't see any problem in it .

**Comment by NikhilM98** [comment]
Screenshot from 2019-03-23 15-52-48

Yeah, when I tried to review, it appeared correctly so I removed the review.
It appears like this in my `Files Changed` tab

**Comment by ashish0910** [comment]
@llaske it is resolved with latest dev branch now . I shall close this pr .

**Comment by ashish0910** [comment]
@llaske in the latest dev branch it is not working now :
Current : 
Screenshot 2019-04-03 at 10 40 30 PM
Screenshot 2019-04-03 at 10 40 42 PM

Expected : 
Screenshot 2019-04-03 at 10 39 06 PM
Screenshot 2019-04-03 at 10 39 14 PM

**Comment by NikhilM98** [comment]
> In the latest dev branch it is not working now

Yes, it was never fixed at the first place. If you test the `dev` branch following the steps you followed when creating first video, it gives false sense that the issue is fixed (Maybe that's why the PR was closed) but the string for `admin` was never translated.

**Comment by llaske** [comment]
I don't see any difference between the last dev branch without the issue. Could you do a video to show the difference?

**Comment by ashish0910** [comment]
Sure @llaske 
Without Fix:

current

After Fix:

expected

**Comment by llaske** [comment]
Ok I see. Thanks.

**Pull request merged**